src/main/java/core/basesyntax/ConstructFigure.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public interface ConstructFigure {
+
+    abstract double getArea();
```

Comment:
it's not getting but calculating
```suggestion
    abstract double calculateArea();
```